### PR TITLE
Setup changes

### PR DESCRIPTION
**Pull request by sebkouba**
Hi Ken,

I'm clearly very new to trying to contribute on Github so I would welcome feedback. It also means I don't really know what I'm doing. I hope I'm not wasting your time... Regarding this PR:

I assume the boilerplate will mostly be used for development so I thought having `npm start` run in dev mode would be the obvious choice and that is what I've seen in other boilerplates. Is there are reason you're deviating from that "standard"?

I thought logging the server port to the console would be useful. I've also seen that elsewhere.

I kept getting an error message / warning about vertx so I included that last change as well.

Should these changes have been separate pull requests and branches on my end? I'm not really clear on that in practice.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -127,7 +127,7 @@ npm run build
 
 # Start server
 ## Note: You need MongoDB running
-npm start
+npm prod
```
Comment (rndstr) [inline README.md]: This won't work. `npm start` and `npm test` work without `run` (possibly others I don't know of) but `prod` requires `npm run prod`

File: webpack/webpack.config.dev-server.js [hunk 2]
```diff
@@ -65,6 +65,7 @@ module.exports = {
         new webpack.DefinePlugin({
           __DEVCLIENT__: false,
           __DEVSERVER__: true
-        })
+        }),
+      new webpack.IgnorePlugin(/vertx/)
```
Comment (rndstr) [inline webpack/webpack.config.dev-server.js]: I had to add that as well after getting the error

```
WARNING in ../~/es6-promise/dist/es6-promise.js
Module not found: Error: Cannot resolve module 'vertx' in /home/rndstr/dev/tmp/react-webpack-node/node_modules/es6-promise/dist
 @ ../~/es6-promise/dist/es6-promise.js 131:20-30
```

Comment (rndstr) [inline webpack/webpack.config.dev-server.js]: And you also need this in the prod server-side rendering part of the config

Comment (sebkouba) [inline webpack/webpack.config.dev-server.js]: I didn't get any errors after adding it to the dev-server. You mean it should go into "webpack.config.prod.js" as well?

File: README.md (hunk 1 above)
Comment (sebkouba) [inline README.md]: Thanks a lot for pointing that out! I was wondering why the commands differ in how they work.

File: webpack/webpack.config.dev-server.js (hunk 2 above)
Comment (rndstr) [inline webpack/webpack.config.dev-server.js]: Exactly. If you run `npm run build:prod` you will also receive the `vertx` error

**Comment by sebkouba** [comment]
I've kept reading realise at least some of my mistakes. I shall create branches in the future...

**Pull request closed without merging**